moonbeam-types-bundle/index.ts

```diff
@@ -336,6 +337,45 @@ const TYPES_800_undefined: RegistryTypes = {
   MultiLocation: "MultiLocationV1",
 };
 
+const TYPES_900_undefined_deprecated: RegistryTypes = {
+  ...TYPES_800_899,
+  MoonbaseRuntimeAssetRegistrarMetadata: {
+    name: "Vec<u8>",
+    symbol: "Vec<u8>",
+    decimals: "u8",
+    isFrozen: "bool",
+  },
+  PalletCrowdloanRewardsRewardInfo: {
+    total_reward: "Balance",
+    claimed_reward: "Balance",
+    contributed_relay_addresses: "Vec<RelayChainAccountId>",
+  },
+  ParachainStakingNominator2: {
+    nominations: "Vec<Bond>",
+    revocations: "Vec<AccountId>",
+    total: "Balance",
+    scheduled_revocations_count: "u32",
+    scheduled_revocations_total: "Balance",
+    status: "NominatorStatus",
+  },
+  ParachainStakingExitQ: {
+    candidates: "Vec<AccountId>",
+    nominators_leaving: "Vec<AccountId>",
+    candidate_schedule: "Vec<(AccountId, RoundIndex)>",
+    nominator_schedule: "Vec<(AccountId, Option<AccountId>, RoundIndex)>",
+  },
+  ParachainStakingCollator2: {
```

Comment:
if this is exactly equal to the `Collator2` type, why not just set this new type equal to that type i.e. `ParachainStakingCollator2: Collator2` and the same for the other types.

Reply:
Yes, I should have done that, but we are going to change the Collator2 to have the camelCase type (see my previous discussion)